From cfed5c76c7b6e56983626799b8d595e64c849d18 Mon Sep 17 00:00:00 2001
From: Reyyzzz24 <revayuliansatria@gmail.com>
Date: Wed, 16 Sep 2026 14:30:38 +0700
Subject: [PATCH 09/25] feat(login): rebuild LoginPoweredBy without
 @rocket.chat/layout (L3a)

ActionLink -> plain <a>, same styling intent. Same useSetting call and
i18n key (registration.page.poweredBy), same admin-togglable visibility.
---
 .../src/components/LoginPoweredBy.tsx             | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/packages/web-ui-registration/src/components/LoginPoweredBy.tsx b/packages/web-ui-registration/src/components/LoginPoweredBy.tsx
index bba93e0..bc8ff72 100644
--- a/packages/web-ui-registration/src/components/LoginPoweredBy.tsx
+++ b/packages/web-ui-registration/src/components/LoginPoweredBy.tsx
@@ -1,5 +1,3 @@
-import { Box } from '@rocket.chat/fuselage';
-import { ActionLink } from '@rocket.chat/layout';
 import { useSetting } from '@rocket.chat/ui-contexts';
 import { Trans } from 'react-i18next';
 
@@ -9,14 +7,19 @@ export const LoginPoweredBy = () => {
 		return null;
 	}
 	return (
-		<Box marginBlockEnd={18}>
+		<div className='ep-mb-4 ep-text-p1'>
 			<Trans i18nKey='registration.page.poweredBy'>
 				{'Powered by '}
-				<ActionLink href='https://rocket.chat/' target='_blank' rel='noopener noreferrer'>
+				<a
+					className='ep-font-medium ep-underline-offset-4 hover:ep-underline'
+					href='https://rocket.chat/'
+					target='_blank'
+					rel='noopener noreferrer'
+				>
 					Rocket.Chat
-				</ActionLink>
+				</a>
 			</Trans>
-		</Box>
+		</div>
 	);
 };
 
-- 
2.50.1 (Apple Git-155)

